src/screens/SurveyDetail/question.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import WheelSelect from 'components/WheelSelect';
+
+const Questions = () => {
+  const question = 'How fulfilled did you feel during this WFH period?';
+  const data = [
+    {
+      display: 'Very fulfilled',
+      value: 0,
+    },
+    {
+      display: 'Somewhat fulfilled',
+      value: 1,
+    },
+    {
+      display: 'Somewhat fulfilled',
+      value: 1,
+    },
+  ];
+
+  return (
+    <div className="questions">
+      <div className="questions__header">
+        <div className="questions__header-right">
+          <button className="button button--close">x</button>
+        </div>
+      </div>
+
+      <div className="questions__container">
+        <div className="questions__details">
+          <div className="questions__number">1/5</div>
+          <h1 className="questions__title"> { question }</h1>
+          <WheelSelect data={ data } multipleSelect={ true } />
+        </div>
+      </div>
+
+      <div className="questions__footer">
+        <div className="questions__footer-right">
+          <button className="button button--arrow">
```

Comment:
This button doesn't seem to show up 🤔  I think the class name should be `button button--circle button--circle-right` instead
![Screen Shot 2563-11-11 at 17 45 02](https://user-images.githubusercontent.com/1772999/98802472-f2378400-2445-11eb-8103-269881ff6332.png)